### PR TITLE
Add steps on unattended upgrades in load balanced setups

### PR DESCRIPTION
**Pull request by sofietoft**

### CODE REVIEW

File: Getting-Started/Setup/Upgrading/general.md [hunk 1]
```diff
@@ -141,6 +147,17 @@ The Runtime level will use `Run` instead of `Upgrade` in order to allow the webs
 The upgrade is run after Composers but before Components. This is because the migration requires services that are registered in Composers and Components requires that Umbraco and the database is ready.
 :::
 
+### Unattended upgrades in a load balanced setup
+
+Follow the steps outlined below to use run unattended upgrades in a load balanced setup.
+
+1. Upgrade Umbraco via NuGet in Visual Studio. Make sure the `Umbraco.Core.ConfigurationStatus` key in `appSetting` in the `web.config` file is updated to match the **target version**.
+2. Deploy to all environments - be sure to include the `appSetting`.
+3. Set the `Umbraco.Core.RuntimeState.UpgradeUnattended` key in `appSetting` in the `web.config` to `true` for **only the Main server**.
```
Comment (rasmusjp) [inline Getting-Started/Setup/Upgrading/general.md:156]:
```suggestion
3. Set the `Umbraco.Core.RuntimeState.UpgradeUnattended` key in `appSetting` in the `web.config` to `true` for **the Main server only**.
```

File: Getting-Started/Setup/Upgrading/general.md [hunk 2]
```diff
@@ -141,6 +147,17 @@ The Runtime level will use `Run` instead of `Upgrade` in order to allow the webs
 The upgrade is run after Composers but before Components. This is because the migration requires services that are registered in Composers and Components requires that Umbraco and the database is ready.
 :::
 
+### Unattended upgrades in a load balanced setup
+
+Follow the steps outlined below to use run unattended upgrades in a load balanced setup.
+
+1. Upgrade Umbraco via NuGet in Visual Studio. Make sure the `Umbraco.Core.ConfigurationStatus` key in `appSetting` in the `web.config` file is updated to match the **target version**.
+2. Deploy to all environments - be sure to include the `appSetting`.
```
Comment (rasmusjp) [inline Getting-Started/Setup/Upgrading/general.md:155]:
```suggestion
2. Deploy to all environments, including the updated `appSetting` for `Umbraco.Core.ConfigurationStatus`.
```

**Review by rasmusjp — CHANGES_REQUESTED** [review]
This is great @sofietoft 💪

I've added a couple of wording suggestions

**Comment by sofietoft** [comment]
Great suggestions @rasmusjp ! I've applied them.

I'll merge this is in as well. Thanks for the review!

**Pull request merged**